go.mod

```
@@ -1,8 +1,8 @@
 module github.com/k0sproject/bootloose
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.4
+toolchain go1.23.3
```

```suggestion
go 1.23
```

Can we omit the toolchain statement?